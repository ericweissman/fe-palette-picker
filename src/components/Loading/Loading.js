import React from 'react'

export const Loading = () => {
  return (
    <div className="loading">
      <h3>loading...</h3>
    </div>
  )
}

export default Loading