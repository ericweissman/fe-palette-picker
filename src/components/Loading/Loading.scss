.loading {
  position: absolute;
  top: 20%;
  left: 45%;
}