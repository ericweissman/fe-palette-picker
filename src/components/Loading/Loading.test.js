import React from 'react';
import { shallow } from 'enzyme';
import { Loading } from './Loading'


describe('Loading', () => {
  let wrapper

  it('should match the initial snapshot', () => {
    wrapper = shallow(<Loading />)

    expect(wrapper).toMatchSnapshot()
  })
})