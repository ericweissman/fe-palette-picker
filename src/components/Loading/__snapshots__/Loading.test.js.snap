// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the initial snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3>
        loading...
      </h3>,
      "className": "loading",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "loading...",
      },
      "ref": null,
      "rendered": "loading...",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3>
          loading...
        </h3>,
        "className": "loading",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "loading...",
        },
        "ref": null,
        "rendered": "loading...",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
